tests/vkray/raygen-trace-ray-param-struct.slang

```diff
@@ -9,9 +9,6 @@ struct testStruct
 
 RWStructuredBuffer<int> outputBuffer1;
 
-// CHECK-DAG: TraceRay(
-// CHECK-DAG: TraceMotionRay(
-
 // CHECK-DAG: void main(
```

Comment:
Can you remove "-DAG"?

Reply:
I can remove them now (since we use ForceInline the code is predictable now)